CONTRIBUTING.md

```diff
@@ -159,22 +144,30 @@ Before creating your pull request, please make sure to review the `tedana` [styl
 
 ### 5. Test your changes
 
-#### Changes to code
+You can run style checks by running the following
 
-For changes to the codebase, we suggest using our development Docker container which will run all the necessary checks and tests to ensure your code is ready to be merged into `tedana`!
-(This does require that you have a local install of [Docker](https://www.docker.com/products/docker-desktop).)
-You can run all the checks with:
+```
+cd TEDANADIR
```

Comment:
Can you explain that `TEDANADIR` is a file path, here ? And, can we mark it as a variable with a `$` ?